framework/include/base/MooseVariableBase.h

```diff
@@ -106,6 +113,8 @@ class MooseVariableBase
 protected:
   /// variable number (from libMesh)
   unsigned int _var_num;
+  /// The FEType associated with this variable
+  FEType _fe_type;
```

Comment:
Likewise, we can probably just store a const reference to this as well. I realize that we had a copy of it in `MooseVariable` but we may as well fix it.